packages/algob/src/lib/algo-operator.ts

```diff
@@ -60,4 +69,45 @@ export class AlgoOperatorImpl implements AlgoOperator {
       confirmedRound: txConfirmation[confirmedRound]
     };
   }
+
+  async deployASC (name: string, scParams: object, flags: ASCDeploymentFlags, payFlags: ASCPaymentFlags
+  ): Promise<ASCInfo> {
+    console.log("Deploying ASC:", name);
+    const result: ASCCache = await this.ensuredCompiled(name, false);
+    const programb64 = result.compiled;
+    const program = new Uint8Array(Buffer.from(programb64, "base64"));
+    const lsig = algosdk.makeLogicSig(program, scParams);
+
+    const params = await tx.getSuggestedParamsWithUserDefaultsASC(this.algodClient, payFlags);
+
+    // ASC1 signed by funder
+    lsig.sign(flags.funder.sk);
+    const funder = flags.funder.addr;
+    const contractAddress = lsig.address();
+
+    // Fund smart contract
+    console.log("Funding Contract:", contractAddress);
+    const encoder = new TextEncoder();
+    const tran = algosdk.makePaymentTxnWithSuggestedParams(funder, contractAddress,
+      flags.microAlgo, payFlags.closeToRemainder,
+      payFlags.note ? encoder.encode(payFlags.note) : undefined,
+      params);
+
+    const signedTxn = tran.signTxn(flags.funder.sk);
+
+    const tranInfo = await this.algodClient.sendRawTransaction(signedTxn).do();
+
+    const confirmedTxn = await this.waitForConfirmation(tranInfo.txId);
+    return {
+      creator: flags.funder.addr,
+      contractAddress: contractAddress,
+      txId: tranInfo.txId,
+      logicSignature: lsig,
+      confirmedRound: confirmedTxn[confirmedRound]
+    };
+  }
+
+  private async ensuredCompiled (name: string, force: boolean): Promise<ASCCache> {
+    return await this.compileOp.ensureCompiled(name, false);
```

Comment:
change `false` to `force`. Let's pass here as method expects.

Reply:
ok